db/organizations/zoominfo.eno

```diff
@@ -0,0 +1,10 @@
+
+name: ZoomInfo
+website_url: https://www.zoominfo.com
+privacy_policy_url: https://www.zoominfo.com/legal/privacy-policy
+privacy_contact: privacy@zoominfo.com
+country: US
+description: ZoomInfo collects business contact and similar information related to individuals when they are working in their professional or employment capacity and uses this information to create professional profiles of individuals and profiles of businesses. We provide this information to our customers, who are businesses trying to reach business professionals for their own business-to-business sales, marketing, and recruiting activities.
```

Comment:
Perhaps we can rephrase it to get rid of it being a self-description ("We provide this information...")?

Reply:
Agreed. I will edit.